recipes/python-logstash/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "python-logstash" %}
+{% set version = "0.4.6" %}
+
+ package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+ source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 10943e5df83f592b4d61b63ad1afff855ccc8c9467f78718f0a59809ba1fe68c
+
+ build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

No need to include the arguments, they're here as default now.

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```